### PR TITLE
Update package descriptions

### PR DESCRIPTION
**Pull request by bmorelli25**
### Summary

This PR updates the package `manifest.yml` descriptions for every top-level package in the integrations repo. The goal of this PR is to add more descriptive descriptions as these will appear as a sub-heading on each integration's documentation page. In this first attempt, I've taken the following approach:

```
This Elastic integration collects + {logs/metrics} + from {integration name}
```

Open to any and all feedback on this as I'm not sure how these changes impact packages elsewhere (like in Kibana). Also, `collecting` is only one part of an integration. Do we want to add other verbs like `parses` or `visualizes`?

### Other integrations

As far as I can tell, this PR updates all relevant integrations except APM, Endpoint, and Symantec -- which I presume live in other repos. I know where APM lives, and I think I can figure out where Endpoint and Symantec live.

### Context

This is related to conversations in https://github.com/elastic/wordlake/pull/1 (sorry, internal link)

### CODE REVIEW

File: packages/cef/manifest.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ name: cef
 title: CEF
 version: 0.3.0
 release: experimental
-description: CEF Integration
+description: This integration collects logs from common event format (CEF) instances
```
Comment (andrewkroh) [inline packages/cef/manifest.yml:5]: CEF isn't a product that you would run an "instance" of. It's a log format like JSON. So perhaps there is a better phrasing of this description.

Comment (bmorelli25) [reply packages/cef/manifest.yml:5]: Thanks, Andrew! I pulled "CEF instance" from L17 of the manifest file. Should I update that line as well to remove it? https://github.com/elastic/integrations/pull/1364/files#diff-966e13402568cef1412dea521600e7fb5172601510e51030c0bbb41d55351e43L17

Comment (bmorelli25) [reply packages/cef/manifest.yml:5]: Not sure if that link will work. Here's a better one: https://github.com/elastic/integrations/blob/a5e8dd252750dcfb3e1f55a65e184c41fd0f643a/packages/cef/manifest.yml#L17

File: packages/auditd/manifest.yml [hunk 2]
```diff
@@ -2,7 +2,7 @@ name: auditd
 title: Auditd
 version: 1.0.0
 release: ga
-description: Auditd Integration
+description: This Elastic integration collects and parses logs from the Audit daemon (`auditd`)
```
Comment (goodroot) [inline packages/auditd/manifest.yml:5]: We will not be able to interpolate the backticks into code snippets.

Suggestion is to remove. 
```suggestion
description: This Elastic integration collects and parses logs from the Audit daemon (auditd)
```

File: packages/log/manifest.yml [hunk 3]
```diff
@@ -2,7 +2,7 @@ format_version: 1.0.0
 name: log
 title: Custom logs
 description: >
```
Comment (goodroot) [inline packages/log/manifest.yml:4]: This "works", but the > is removed when it's parsed.

File: packages/system/manifest.yml [hunk 4]
```diff
@@ -3,7 +3,7 @@ name: system
 title: System
 version: 1.0.0
 license: basic
-description: System Integration
+description: This Elastic integration collects logs and metrics of your Systems
```
Comment (goodroot) [inline packages/system/manifest.yml:6]: This blew my mind. Maybe elaborate a bit on systems?

Comment (bmorelli25) [reply packages/system/manifest.yml:6]: LOL. Deal.

File: packages/auditd/manifest.yml (hunk 2 above)
Comment (bmorelli25) [reply packages/auditd/manifest.yml]: Perfect. I was wondering about that.

File: packages/log/manifest.yml (hunk 3 above)
Comment (bmorelli25) [reply packages/log/manifest.yml:4]: 👍 Should be OK then. I'm not sure why some descriptions use this format -- so I chose not to mess with it.

File: packages/apache/manifest.yml [hunk 5]
```diff
@@ -3,7 +3,7 @@ name: apache
 title: Apache
 version: 0.8.1
 license: basic
-description: Apache Integration
+description: This Elastic integration collects logs and metrics from Apache servers
```
Comment (mtojek) [inline packages/apache/manifest.yml:6]: Integrations (or packages in general) are idempotent, which mean that you have bump up the version (I suggest semver's "minor") and add an entry to the changelog file.

Unfortunately you have to do this for every integration.

File: packages/elastic_agent/manifest.yml [hunk 6]
```diff
@@ -2,8 +2,7 @@ name: elastic_agent
 title: Elastic Agent
 version: 0.0.7
 release: experimental
-description: Elastic Agent Integration
-type: integration
```
Comment (mtojek) [inline packages/elastic_agent/manifest.yml:6]: Please revert this line (`type: integration`). You may notice that CI complained about it.

**Review by mtojek — CHANGES_REQUESTED** [review]
In general we use `elastic-package format` to format sources of integrations.

File: packages/elastic_agent/manifest.yml (hunk 6 above)
Comment (bmorelli25) [reply packages/elastic_agent/manifest.yml:6]: Oops! Good catch. This was an accident.

File: packages/apache/manifest.yml (hunk 5 above)
Comment (bmorelli25) [reply packages/apache/manifest.yml:6]: 😬 OH

Comment (bmorelli25) [reply packages/apache/manifest.yml:6]: 😄 Thanks, will get started

Comment (bmorelli25) [reply packages/apache/manifest.yml:6]: Not as bad as I thought it would be. Updated!

**Review by mtojek — CHANGES_REQUESTED** [review]
@bmorelli25 

We have requirement that CI must report green state to get the PR merged. In this case is red which means there is something wrong with this PR. There are problems with elastic_agent (missing manifest entry) and okta (missing manifest entry).

File: packages/elastic_agent/changelog.yml [hunk 7]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "0.1.7"
```
Comment (bmorelli25) [inline packages/elastic_agent/changelog.yml:2]:
```suggestion
- version: "0.1.0"
```

File: packages/okta/manifest.yml [hunk 8]
```diff
@@ -2,9 +2,9 @@ name: okta
 title: Okta
 version: 1.0.1
 release: ga
-description: Okta Integration
+description: This Elastic integration collects events from Okta
 type: integration
-format_version: 1.0.0
+format_version: 1.1.0
```
Comment (bmorelli25) [inline packages/okta/manifest.yml:7]:
```suggestion
format_version: 1.0.0
```

File: packages/okta/manifest.yml [hunk 9]
```diff
@@ -2,9 +2,9 @@ name: okta
 title: Okta
 version: 1.0.1
```
Comment (bmorelli25) [inline packages/okta/manifest.yml:3]:
```suggestion
version: 1.1.0
```

**Comment by bmorelli25** [comment]
> There are problems with elastic_agent (missing manifest entry) and okta (missing manifest entry).

Thanks! I _think_ everything should pass this time 🤞

**Comment by bmorelli25** [comment]
Noooooo. Green, but a change was merged to the system integration. I'll fix the conflicts and let the tests run again.

**Review by mtojek — APPROVED** [review]
Remember to bump up package versions accordingly in case of conflicts.

LGTM, feel free to merge this PR if CI is happy.

**Comment by bmorelli25** [comment]
> Remember to bump up package versions accordingly in case of conflicts.

Yup. The merged update in this case happened to be a patch, so my minor update was still the latest -- just needed to merge in the new CL entry.

> LGTM, feel free to merge this PR if CI is happy.

Thanks! I'll be clicking the button the second it's green :)

**Comment by bmorelli25** [comment]
Test just timed out. Restarting.

**Pull request merged**